source/2020-05-22-the-ember-times-issue-149.md

```diff
@@ -0,0 +1,146 @@
+---
+title: The Ember Times - Issue No. XXS
+author: Matthew Roderick, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/xx/xx-the-ember-times-issue-XXX.html"
+responsive: true
+---
+
+<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+
+READMORE
+
+---
+
+## [Cal Woolgar's Video Series 🐺](https://www.youtube.com/channel/UCVe_D9xXXDwXyU2o0_cadxA)
+
+In this video series, Cal Woolgar's first video 🎥 [What is Ember.js?](https://www.youtube.com/watch?v=jkb1At1pk2E) breaks down the basics of [Ember.js.](https://emberjs.com/) 
+
+ This first video in his series mentions things like [Handlebars Templating language](https://handlebarsjs.com/), the separation from Java and HTML, [Ember cli](https://ember-cli.com/), and how easy it is to create a new application.
+
+
+ [Cal](https://twitter.com/callam_woolgar) does not go too far into these things in his first video. **He aims to make videos short and sweet 🍭** so that someone learning can come back and easily find something without having to go through hours of video. This series is excellent so far, and we look forward to what's next from Cal. 👏
```

Comment:
```suggestion
## ["My Experience with Ember.js" video series 🐺](https://www.youtube.com/watch?v=jkb1At1pk2E&t=)

[Cal Woolgar (@calWoolgar)](https://github.com/calWoolgar) has kicked off a new video series "My Experience with Ember.js", where he breaks down the basics of Ember. 

The first video [What is Ember.js?](https://www.youtube.com/watch?v=jkb1At1pk2E) explains the [Handlebars templating language](https://handlebarsjs.com/) and how it separates your JavaScript from HTML. Cal also touches on [ember-cli](https://ember-cli.com/), and how easy it is to create a new application.

Cal aims to make his videos **short and sweet** 🍭 so that someone learning can come back and easily find something in bite-sized pieces. Look forward to what's next from Cal! 👏
```

- link you had didn't link to the series, but his youtube page which has a bunch of non-ember content
- we typically refer to the github handle, but if there's a relevant twitter link that could be added!

Reply:
i brought in these copy edits. let's make further edits (if any) in a new branch, since things get out of date quickly